bin/tomcat-ctl

```diff
@@ -172,6 +172,7 @@ tomcat_start () {
         cd /git-clones/katt-kfs-rhubarb/ && bundle install
         cd /git-clones/katt-kfs-rhubarb/ && cap local deploy
         cd $startdir
+        touch /transactional/$HOSTNAME.runnable
```

Comment:
From within the docker container, the path for the .runnable file needs to be /transactional/work/control/:

root@c8732c3a7b81:/data# ls -l /transactional/work/control/
b3c8672d8e19.runnable  history/